sequencer/src/options.rs

```diff
@@ -188,6 +188,16 @@ pub struct Options {
     #[derivative(Debug(format_with = "fmt_urls"))]
     pub state_peers: Vec<Url>,
 
+    /// Peer nodes use to fetch missing config
+    ///
+    /// Typically, the network-wide config is fetched from the orchestrator on startup and then
+    /// persisted and loaded from local storage each time the node restarts. However, if the
+    /// persisted config is missing when the node restarts (for example, the node is being migrated
+    /// to new persistent storage), it can instead be fetched directly from a peer.
+    #[clap(long, env = "ESPRESSO_SEQUENCER_CONFIG_PEERS", value_delimiter = ',')]
+    #[derivative(Debug(format_with = "fmt_opt_urls"))]
+    pub config_peers: Option<Vec<Url>>,
```

Comment:
Is there any downside to changing this to `Vec<Url>` and then treating an empty vector as nothing? It may remove some complexity

Reply:
I personally prefer `Option<Vec>` when empty is qualitatively different from non-empty because the compiler forces you to handle the `None` case explicitly whenever you want to use the vector. Whereas with just `Vec<Url>`, it's pretty easy to forget to check `is_empty` which is bad when there's entirely separate logic you want to do in the empty case